source/postcard-rpc/Cargo.toml

```diff
@@ -36,7 +36,7 @@ postcard-schema = { version = "0.2.1", features = ["derive"] }
 #
 
 [dependencies.nusb]
-version = "0.1"
+version = "0.1.9"
```

Comment:
I'm interested if this update was for a specific reason!

Reply:
The `Interface::descriptors` API was added in 0.1.9 hence the bump